.gitlab-ci.yml

```diff
@@ -25,6 +24,44 @@ unit_tests_julia1.9:
   tags:
     - cpuonly
 
+unit_tests_releases:
+  extends: .untit_test_template
+  parallel:
+    matrix:
+      - JULIA_VERSION: ["1.6", "1.7", "1.8", "1.9"]
+  image: julia:$JULIA_VERSION
+
+unit_tests_nightly:
+  extends: .untit_test_template
+  # use the same baseimage like the official julia images
+  image: debian:bookworm-slim
```

Comment:
Is this the base image, which is also used for `julia:$JULIA_VERSION` above?

Reply:
Yes, see here: https://hub.docker.com/layers/library/julia/1.9.3/images/sha256-44b6dfeb1646a41b5de39294740ba603b6d18164e6077c5e2bb5b64677b41f74?context=explore

Reply:
Nice! Thanks for clarifying.